src/middlewared/middlewared/plugins/chart_releases_linux/redeploy.py

```diff
@@ -0,0 +1,45 @@
+import errno
+import os
+
+from middlewared.schema import accepts, Str, Ref, returns
+from middlewared.service import CallError, job, Service
+
+from .utils import add_context_to_configuration, CONTEXT_KEY_NAME, get_action_context
+
+
+class ChartReleaseService(Service):
+
+    class Config:
+        namespace = 'chart.release'
+
+    @accepts(Str('release_name'))
+    @returns(Ref('chart_release_entry'))
+    @job(lock=lambda args: f'chart_release_redeploy_{args[0]}')
+    async def redeploy(self, job, release_name):
+        """
+        Redeploy will initiate a rollout of new pods according to upgrade strategy defined by the chart release
+        workloads. A good example for redeploying is updating kubernetes pods with an updated container image.
```

Comment:
slight nitpick: redeploy is technically not limited to pods, which might be confusing for people that do not know it calles `helm update` in the end.

```suggestion
        Redeploy will initiate a new rollout of the Helm chart according to upgrade strategy defined by the chart release
        workloads. A good example for redeploying is updating kubernetes pods with an updated container image.
```

Reply:
Thanks for pointing, addressed.